### DISCUSSION TITLE
Romm self hosted rom manager

### DISCUSSION THREAD

**Discussion by prvsmrtsltppr7** [tteck/Proxmox#3468] [Ideas]
[RomM](https://github.com/rommapp/romm) (ROM Manager) allows you to scan, enrich, and browse your game collection with a clean and responsive interface. With support for multiple platforms, various naming schemes, and custom tags, RomM is a must-have for anyone who plays on emulators.

A script to automate this would be great.

**Comment by SamBudeh** [comment]
+1 for RomM.
